autoload/neomake/makers/ft/rst.vim

```diff
@@ -23,3 +23,19 @@ function! neomake#makers#ft#rst#rstcheck()
             \ '%E%f:%l: (SEVERE/4) %m',
         \ }
 endfunction
+
+" TODO: determine proper path of current .rst file upwards in sear for conf.py file, use that instead of "docs"
+" TODO: ask noomake for a temporary folder instead of /tmp/neomake-sphinx
```

Comment:
It's just a directory where sphinx will create multiple .xml files. There are no options to suppress that.

does not really matter where it is, can be in current project or elsewhere. 

https://github.com/vim-syntastic/syntastic/blob/master/syntax_checkers/rst/sphinx.vim#L25